### PR TITLE
Review stats

### PR DESCRIPTION
**Pull request by evilaspaas1**
Add feature to display a users best and worst reviews on their show page

- Add model tests for best and worst reviews
- Add Test to display a users best and worst reviews
- Update pet4 description
- Update edge case for worst review
- Update average review rating method to account for nil value. Add best and worst review method
- Add best and worst review to users show page

### CODE REVIEW

File: app/models/user.rb [hunk 1]
```diff
@@ -8,6 +8,18 @@ class User < ApplicationRecord
   has_many :shelters, through: :reviews
 
   def average_review_rating
-    reviews.average(:rating)
+    avg = reviews.average(:rating)
```
Comment (c-bartell) [inline app/models/user.rb:11]: Great job accounting for this edge case!

File: app/views/users/show.html.erb [hunk 2]
```diff
@@ -7,6 +7,17 @@
 
 <%= content_tag :h2, "User Reviews:" %>
 <%= content_tag :h4, "Average Review Rating: #{@user.average_review_rating.to_f.round(1)}" %>
+<section id="highlighted">
+  <% if @user.best_review && @user.reviews.count > 1%>
```
Comment (c-bartell) [inline app/views/users/show.html.erb:11]: I think this is a great example of appropriate display logic.

**Review by c-bartell — APPROVED** [review]
Great job!
